sources/proto/Api.pbobjc.h

```diff
@@ -365,7 +365,7 @@ typedef GPB_ENUM(ITMSavedArrangementRequest_Action) {
   /** Restore an existing arrangement with the given name */
   ITMSavedArrangementRequest_Action_Restore = 0,
 
-  /** Save windows to a new arrangment with the given name */
+  /** Save windows to a new arrangement with the given name */
```

Comment:
This file is autogenerated. Don't make any changes to it; instead change api.proto and this will get updated next time it's built.